sycl/source/detail/os_util.cpp

```diff
@@ -170,7 +170,7 @@ std::string OSUtil::getDirName(const char* Path) {
   std::string Tmp(Path);
   // dirname(3) needs a writable C string: a null-terminator is written where a
   // path should split.
-  size_t TruncatedSize = strlen(dirname(const_cast<char*>(Tmp.c_str())));
+  size_t TruncatedSize = strlen(dirname(const_cast<char *>(Tmp.c_str())));
```

Comment:
This is to avoid this:
error: implicit conversion between pointer-to-function and pointer-to-object is a Microsoft extension [-Werror,-Wmicrosoft-cast]

Reply:
The change is simply adding a space into `char *`. Not sure how this leads to the described effect. Am I missing something?

Reply:
This comment should have been placed at line#199 below.